frontend/webapp/app/layout.tsx

```diff
@@ -8,6 +8,8 @@ const LAYOUT_STYLE: React.CSSProperties = {
   margin: 0,
   position: "fixed",
   scrollbarWidth: "none",
+  width: "100vw",
+  height: "100vh",
```

Comment:
Here you use `100vw` / `100vh` but below you use `100%`. is it intentional?

Reply:
The entire screen will occupy 100vw, the relative part of the page (e.g overview), will be 100% of the screen remaining after the width of the side menu, so will the other pages